README.md

```diff
@@ -36,7 +36,17 @@ Each branch should satisfy this mask `{KEYWORD}/{TICKET_NUMBER}`. For example: `
 ### Git flow
 ![git flow](src%2Fassets%2FUML%2Fimages%2Fgit%20flow.JPG)
 
-#### Main points:
-- unused branches have to be deleted
-- need to squash commits before merge to dev
-- CI\CD is looking on main\dev branches, and new version will be built only if new code is pushed to main branch
+## Setup project
```

Comment:
### ❗ Issue: Move setup project part

This is important thing and should be at the top of this page. Maybe just before `git conventions`